domene/src/main/java/no/nav/foreldrepenger/mottak/domene/oppgavebehandling/OpprettGSakOppgaveTask.java

```diff
@@ -114,15 +120,15 @@ private String opprettOppgave(ProsessTaskData prosessTaskData, BehandlingTema be
         // Oppgave har ikke mapping for alle undertyper fødsel/adopsjon
         final String brukBT = BehandlingTema.forYtelseUtenFamilieHendelse(behandlingTema).getOffisiellKode();
 
-        String arkivId = prosessTaskData.getPropertyValue(ARKIV_ID_KEY);
+        String journalpostId = prosessTaskData.getPropertyValue(ARKIV_ID_KEY);
 
         // Overstyr saker fra NFP+NK, deretter egen logikk hvis fødselsnummer ikke er
         // oppgitt
-        final String enhetId = enhetsidTjeneste.hentFordelingEnhetId(Tema.FORELDRE_OG_SVANGERSKAPSPENGER, behandlingTema,
+        final String enhetId = enhetsTjeneste.hentFordelingEnhetId(Tema.FORELDRE_OG_SVANGERSKAPSPENGER, behandlingTema,
             Optional.ofNullable(enhetInput), prosessTaskData.getAktørId());
         final String beskrivelse = lagBeskrivelse(behandlingTema, dokumentTypeId, prosessTaskData);
-        return enhetsidTjeneste.opprettJournalføringsOppgave(arkivId, enhetId, prosessTaskData.getAktørId(),
-            prosessTaskData.getPropertyValue(SAKSNUMMER_KEY), brukBT, beskrivelse);
+        return oppgaverTjeneste.opprettJournalføringsoppgaveFor(JournalpostId.fra(journalpostId), enhetId, prosessTaskData.getAktørId(),
+            prosessTaskData.getPropertyValue(SAKSNUMMER_KEY), brukBT, beskrivelse, OppgaveSystem.LOKALT);
```

Comment:
Dette går ikke helt bra vel - alltid LOKAL. Ref regler om 4292 og UDEFINERT/ANNET. 
OppgaveSystem-utlederen trenger 3 deler:
* enhetId = 4292 || Set.of(UDEFINERT,ANNET).contains(dokumentTypeId) -> GOSYS
* dokumentTypeId.erInntektsmelding/erSøknadType/erKlageType -> LOKAL
* default - GOSYS i første omgang
* 
Er det ikke like greit med 2 metoder opprettJournalføringsoppgaveGosysFor og opprettJournalføringsoppgaveLokalFor - i stedet for en enum med det samme.

Reply:
Jobber fortsatt med den :) Gir beskjed når den er klar 👍🏻

Reply:
Klar nå